platform/src/test/kotlin/org/stellar/anchor/platform/config/Sep10ConfigTest.kt

```diff
@@ -201,52 +224,4 @@ class Sep10ConfigTest {
     config.postConstruct()
     assertEquals("localhost:8080", config.webAuthDomain)
   }
-
-  companion object {
-    @JvmStatic
-    fun validOmnibusAccounts(): Stream<List<String>> {
```

Comment:
Are these functions not used?

Reply:
Nope, I think it was left there unintentionally. We can add it back if we need it in the future.